requirements/pyproject.toml

```diff
@@ -24,7 +24,7 @@ ckanext-qa = {git = "https://github.com/ckan/ckanext-qa.git" }
 ckanext-saml2auth = { git = "https://github.com/keitaroinc/ckanext-saml2auth.git", tag = "ckan-2.8" }
 ckanext-dcat = {git="https://github.com/ckan/ckanext-dcat" }
 webob = {git = "https://github.com/GSA/webob.git", branch = "ckan-patch" }
-pygments = {git = "https://github.com/GSA/pygments.git", branch = "datagov/v2.2.x" }
+pygments = {git = "https://github.com/GSA/pygments.git", branch = "feature/redos-fix" }
```

Comment:
Did we want to force-push this to `datagov/v2.2.x`?

Reply:
We could force push this to a new `datagov/v2.5.x`, but with our recent focus of getting onto python3 this will hopefully go away soon. I'm going to leave it as is for now.

Reply:
python3 is important, but it's not an excuse to break our conventions or compliance processes. compliance requires that we use protected branches, for example.